### DISCUSSION TITLE
Plotting Result of NSE on Pyvista from OpenFoam

### DISCUSSION THREAD

**Discussion by dbenjamin28** [pyvista/pyvista#6376] [Q&A]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
None
```

### Description

I want to model the air flow around a moving wind using NSE. I want to capture the pressure and the velocity of the wind. 

### System Information

```shell
Date: Sat Jul 13 11:01:30 2024 Pacific Daylight Time

                OS : Windows (11 10.0.22631 SP0 Multiprocessor Free)
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
       Environment : Python
        GPU Vendor : Intel
      GPU Renderer : Intel(R) Iris(R) Xe Graphics
       GPU Version : 4.5.0 - Build 31.0.101.5186
  MathText Support : False

  Python 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64
  bit (AMD64)]

           pyvista : 0.44.0
               vtk : 9.3.1
             numpy : 2.0.0
        matplotlib : 3.9.1
            scooby : 0.10.0
             pooch : 1.8.2
            pillow : 10.4.0
```

**Comment by tkoyama010** [comment]
@dbenjamin28
Could you go this page and check it is working in your environment?
https://docs.pyvista.org/version/stable/examples/99-advanced/openfoam-example#sphx-glr-examples-99-advanced-openfoam-example-py
You can download script or Jupyter Notebook at the end of the page.
![image](https://github.com/user-attachments/assets/bfde0252-0fc5-46f5-a965-9b56f49fa626)

**Comment by MatthewFlamm** [comment]
There is a full example of this in https://docs.pyvista.org/version/stable/examples/99-advanced/openfoam-example#sphx-glr-examples-99-advanced-openfoam-example-py which demonstrates how to control reading data and then plotting it.

Another example is https://docs.pyvista.org/version/stable/examples/99-advanced/openfoam-tubes#sphx-glr-examples-99-advanced-openfoam-tubes-py. But this one looks to have some bug in the newest release.

**Reply by tkoyama010** [reply]
> Another example is https://docs.pyvista.org/version/stable/examples/99-advanced/openfoam-tubes#sphx-glr-examples-99-advanced-openfoam-tubes-py. But this one looks to have some bug in the newest release.

Seems that is it working in 0.43.
https://docs.pyvista.org/version/0.43/examples/99-advanced/openfoam-tubes.html
@dbenjamin28 Please install pyvista<0.44 if you want to use feature now.
```bash
pip install pyvista<0.44
```

**Reply by MatthewFlamm** [reply]
The problem was with the dataset, and not pyvista plotting of OpenFOAM.  It was fixed in #6380 and is available in pyvista 0.44.1.
